src/styles/main.scss

```
@@ -1,3 +1,55 @@
+html {
+  font-family: Arial, sans-serif;
+  color: white;
+  font-size: 100px;
+}
+
 body {
+  height: 100%;
+  margin: 0;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px,1fr));
+}
+
+@media screen and (min-width: 1200px) {
+  .container {
+    display: grid;
+    grid-template-columns: repeat(3, minmax(300px,1fr));
```

You don't need to set a lower value because your cell should take 33% percent of the available width. 
```suggestion
    grid-template-columns: repeat(3, 1fr);
```

---

✓